src/Command/ConsumerCommand.php

```diff
@@ -51,13 +54,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $provider = $broker->getProvider($gate);
         $producer = $broker->getProducer($gate);
 
-        $processor = new CommandProcessor($logger, new ProcessBuilder,
-                                          $provider, $producer,
-                                          $container->getRootDir() . '/app/console',
-                                          $kernel->getEnvironment(),
-                                          true === $input->getOption('disable-verbosity-propagation')
-                                            ? OutputInterface::VERBOSITY_QUIET
-                                            : $output->getVerbosity()
+        $processor = new CommandProcessor(
+            $logger,
+            new ProcessBuilder,
+            $provider,
+            $producer,
+            sprintf('%s %s', $container->getRootDir(), $consolePath),
```

Comment:
I suppose yes, I change (and I just see an error, should be `'%s%s'` not`'%s %s'`